init_code.py

```diff
@@ -1,65 +1,40 @@
 # This defines the serialization/deserialization code for keras models when used
 # in deepforge
 
+import os
+import time
+import tarfile
+import shutil
+
+import tensorflow.keras as keras
+
 import deepforge
-import keras
 
-def dump_model(model, outfile):  # Create tar ball of custom_objects and model
+
+def dump_model(model, outfile):
     # Create the tmp directory
-    import os
-    import time
     tmp_dir = outfile.name + '-tmp-' + str(time.time())
-    os.makedirs(tmp_dir)
-
-    # store any custom objects in a file
-    if hasattr(model, 'custom_objects'):
-        co_path = os.path.join(tmp_dir, 'custom_objects')
-        with open(co_path, 'wb') as f:
-            import dill as pickle
-            pickle.dump(model.custom_objects, f)
+    model.save(tmp_dir)
 
-    # write the model to a file
-    model_path = os.path.join(tmp_dir, 'model')
-    model.save(model_path)
-
-    # tar both files together into the outfile
-    import tarfile
     with tarfile.open(outfile.name, 'w:gz') as tar:
-        tar.add(model_path, arcname='model')
-        if hasattr(model, 'custom_objects'):
-            tar.add(co_path, arcname='custom_objects')
+        tar.add(tmp_dir, arcname='SavedModel')
 
-    import shutil
     shutil.rmtree(tmp_dir)
 
+
 def load_model(infile):
-    # Make a tmp dir
-    import os
-    from os import path
-    import time
     tmp_dir = infile.name + '-tmp-' + str(time.time())
```

Comment:
Why hasn't this been removed, too?

Reply:
This will make sure there are no conflicts in the `SavedModel` directory. i.e. deserializing two models in the same directory can cause problems.